src/Router/ContainerAwareRouter.php

```diff
@@ -2,13 +2,16 @@
 
 namespace Bernard\Router;
 
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use Psr\Container\ContainerInterface;
 
 /**
- * @package Bernard
+ * @package Bernard\Router
  */
 class ContainerAwareRouter extends SimpleRouter
 {
+    /**
+     * @var ContainerInterface
+     */
     private $container;
```

Comment:
I saw a comment in some repo that private properties don't have docblock and for consistency purpose, we should do the same.